pkg/flare/archive_win.go

```diff
@@ -100,6 +100,28 @@ func zipTypeperfData(tempDir, hostname string) error {
 	}
 	return nil
 }
+func zipLodctrOutput(tempDir, hostname string) error {
+	cmd := exec.Command("lodctr", "/q")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		log.Warnf("Error running lodctr command %v", err)
```

Comment:
Should we return an error here ? Seems like we're going to create a empty file later. Or do we want to create the `lodctr.txt` file with the error ?

Reply:
I had to re-debug this to remember why I did it this way.  The `lodctr` command seems to return an error code even on success.  So we log the return code and keep going regardless.